website/landingpage/dev/src/pages/blog/overview.html

```diff
@@ -5,3 +5,13 @@
 ---
 
 {{{ blogentries }}}
+
+<div id="blog-chunk"> 
+</div> 
+
+<div class="row" id="blog-actions">
+    <div class="column">
+        <a id="load-more-blogs-btn" href="#blog-chunk" data-smooth-scroll class="btn-primary">Load more blogs</a>
```

Comment:
@alexandra-simeonova might want to have a look at my suggestion :-)
```suggestion
        <a id="load-more-blogs-btn" href="#blog-chunk" data-smooth-scroll class="btn-primary">Load more entries</a>
```

Reply:
I think `Load more entries` is great 👍